onyo/lib/onyo.py

```diff
@@ -203,155 +203,56 @@ def clear_caches(self,
             self.git.clear_caches(files=True)
 
     def generate_commit_message(self,
-                                message: Optional[list[str]] = None,
-                                cmd: str = "",
-                                keys: Optional[list[str]] = None,
-                                destination: Optional[Path] = None,
+                                format_string: str,
                                 max_length: int = 80,
-                                modified: Optional[Iterable[Path]] = None) -> str:
-        """Generate a commit message subject and body suitable for use with
-        `git commit`.
+                                **kwargs) -> str:
+        """Generate a commit message subject suitable for use with `git commit`.
```

Comment:
See section "Returns":

```suggestion
        """Generate a commit message subject.
```